include/roboteam_ai/stp/Play.hpp

```diff
@@ -59,9 +60,9 @@ class Play {
     virtual ~Play() = default;
 
     /**
-     * Default ctor, ensures proper construction of Play
+     * Ctor that constructs a play an assigns its name
```

Comment:
I think I made a typo, oops
```suggestion
     * Ctor that constructs a play and assigns its name
```